src/index.html

```diff
@@ -1,12 +1,358 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="/styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo-logo"
+        >
+      </a>
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              href="index.html"
+              class="header__item-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> Samsung </a>
```

Comment:
IHO, is a little better because we can see the content without reading the whole line
```suggestion
            <a href="index.html" class="header__item-link">
              Samsung
            </a>
```